drivers/npmx/npmx_shell.c

```diff
@@ -1122,8 +1122,9 @@ static int cmd_ldsw_mode_set(const struct shell *shell, size_t argc, char **argv
 	}
 
 	int err = 0;
-	uint8_t ldsw_indx = (uint8_t)shell_strtoul(argv[1], 0, &err);
-	npmx_ldsw_mode_t mode = (npmx_ldsw_mode_t)shell_strtoul(argv[2], 0, &err);
+	uint8_t ldsw_indx = CLAMP(shell_strtoul(argv[1], 0, &err), 0, UINT8_MAX);
+	npmx_ldsw_mode_t mode =
+		(npmx_ldsw_mode_t)(CLAMP(shell_strtoul(argv[2], 0, &err), 0, UINT8_MAX));
```

Comment:
Same as for `npmx_buck_vout_select_t`

Reply:
I think we shouldn't do that, lease take a look at https://github.com/NordicSemiconductor/npmx-zephyr/pull/24#discussion_r1244009996